simulator/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub struct DisplayBuilder {
 impl DisplayBuilder {
     pub fn new() -> Self {
         Self {
-            width: 256,
+            width: 320,
```

Comment:
Why does this need to change?

Reply:
I added triangles to the stroke example and wanted to make them fit into the window. I didn't realize that this should be done in the example itself. Should be fixed with my last commit.